hashes/rust

```diff
@@ -1,6 +1,6 @@
-# https://static.rust-lang.org/dist/rustc-1.56.0-src.tar.xz
-SHA512 (rustc-1.56.0-src.tar.xz) = 2daa365524b47dcc48e49a0e9c8c45988af44c0845e2695dc5053f18e768e49acf3dbdd77f808dbf260546ef608eb47c593544012dd05675cb7e6b6223900315
-### See https://github.com/rust-lang/rust/blob/1.56.0/src/stage0.txt for what to use below. ###
+# https://static.rust-lang.org/dist/rustc-1.56.1-src.tar.xz
+SHA512 (rustc-1.56.1-src.tar.xz) = 193468e211cde9ebc5f6e30b8e3733b79bd8710fe6dd45c7ed8d4392f91010d30466787afd4d0b2041cd7dd994924fee8ad111048824e248bd994959e55bf15f
+### See https://github.com/rust-lang/rust/blob/1.56.1/src/stage0.txt for what to use below. ###
```

Comment:
This is worrying me a bit.  Why wouldn't any of tarballs below change?

Looks like stage0.txt didn't change at all between the 1.56.0 and 1.56.1 trees.  They've gotten rid of stage0.txt in the live tree, switching to stage0.json instead.  I wonder if they're not updating stage0.txt anymore.

I think we should find some way to confirm this is correct, given that (I think) the build process uses the tarballs below, and they didn't change..?

Reply:
The difference between 1.56.0 and 1.56.1 is pretty much a single commit https://github.com/rust-lang/rust/commit/dd61274930ec0cd17711fab52d2bc9ad3e9053de so I wasn't terribly surprised to see the same stage0.txt as the 1.56.0 release.

Reply:
Oh, it's because the tarballs below are the 1.55 tarballs, used as stage0 to build 1.56, and 1.55 hasn't changed.  Just this source tarball of 1.56 would change for a point release.  Same thing happened with the 1.45.x point releases.